backend/routes/room.js

```diff
@@ -39,7 +39,7 @@ function cleanDB() {
 //This part runs the CronJob
 //Currently runs every 4 hours
```

Comment:
Please change the comment as it runs every 12 hours

Reply:
Fixed @savannahar68